### DISCUSSION TITLE
basic/meet-problems/

### DISCUSSION THREAD

**Discussion by giscus** [SAST-skill-docers/sast-skill-docs#61] [Announcements]
# basic/meet-problems/

https://docs.net9.org/basic/meet-problems/

**Comment by XOEDLXH** [comment]
当然是使用deepseek来编程啦（bushi

**Comment by lsmx0** [comment]
哈哈哈

**Comment by Runner0** [comment]
目前倒是可以为了节省学习人员的时间 先用gpt进行所谓的自然语言发问&自然语言检索了；

大多数情况下可以跳到指定的github上某个issues或者reddit/stackoverflow上某个会话；

当然传统的检索式作为基本技能还是必要的；
